.eslintrc.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
 		'@typescript-eslint/no-non-null-assertion': 'off',
 		'@typescript-eslint/no-explicit-any': 'off',
 		'@typescript-eslint/ban-ts-comment': 'off',
+		'@typescript-eslint/method-signature-style': ['error', 'method'],
```

Comment:
idk if you've seen this? i don't know if it's actually any good though ts is weird about `this`: https://typescript-eslint.io/rules/unbound-method/